src/DDTrace/OpenTelemetry/Span.php

```diff
@@ -75,6 +92,29 @@ private function __construct(
             // done in serializer.c:ddtrace_set_root_span_properties (as of v0.92.0)
             $span->name = $this->operationNameConvention = Convention::defaultOperationName($span);
         }
+
+        $this->uniqueIdentifier = \spl_object_hash($this); // traceId + spanId is NOT a valid unique identifier, as the traceId can be modified
+
+        // Set the span links
+        if ($isRemapped) {
+            // At initialization time (now), only set the links if the span was created using the OTel API
+            // Otherwise, the links were already set in DD's OpenTelemetry\Context\Context
+            foreach ($links as $link) {
+                /** @var LinkInterface $link */
+                $linkContext = $link->getSpanContext();
+
+                $spanLink = new SpanLink();
+                $spanLink->traceId = $linkContext->getTraceId();
+                $spanLink->spanId = $linkContext->getSpanId();
+                $spanLink->traceState = (string)$linkContext->getTraceState(); // __toString()
+                $spanLink->attributes = $link->getAttributes()->toArray();
+                $spanLink->droppedAttributesCount = 0; // Attributes limit aren't supported/meaningful in DD
+
+                // Save the link
+                ObjectKVStore::put($spanLink, $this->uniqueIdentifier, $link);
```

Comment:
Maybe I'm missing something, but why an uniqueIdentifier? Isn't the DD SpanLink to OTel SpanLink mapping 1:1? It would be needed if the span links were dependent on the specific span they're attached to, but they aren't? So just using `"OTelLink"` as key would be fine too?